src/Entity/PackageSet.php

```diff
@@ -40,6 +41,13 @@ class PackageSet
 
     protected $id;
     protected $name;
+
+    /**
+     * @Assert\Unique(
+     *  normalizer="AppBundle\Entity\Package::getPackageName",
```

Comment:
Do you do it because there is no way to call `getName` directly?

Reply:
I dont know exactly. "normalizer" arg has to be an anonymous function so I had difficulties to find the correct format so I ended up with a static function.